x/bank/client/rest/sendtx.go

```diff
@@ -20,6 +20,7 @@ import (
 // RegisterRoutes - Central function to define routes that get registered by the main application
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *wire.Codec, kb keys.Keybase) {
 	r.HandleFunc("/accounts/{address}/send", SendRequestHandlerFn(cdc, kb, cliCtx)).Methods("POST")
+	r.HandleFunc("/broadcast", BroadcastTxRequestHandlerFn(cdc, cliCtx)).Methods("POST")
```

Comment:
@jackzampolin do you like just `/broadcast`?  maybe we should change it to be more RESTful ? I'd like to hear some ideas here

Reply:
I'm open to any suggestions!

Reply:
See above